### ISSUE TITLE
Survival Door/chest lock menu

### ISSUE THREAD

**Issue by moepi2k** [suriyun-production/mmorpg-kit-docs#266]
when using controller mode "Combat" you can only open/close door or chest. but you are not able to open seperate menu to unlock or set code. also not with doubleclick like in adventure mode. best thing is to open that menu by long press interact button "E"

**Comment by insthync** [comment]
Oh, I found the source of issues, in `BasePlayerCharacterController.cs` function `ShowCurrentBuildingDialog` changes code in line 236 to
```
            CacheUISceneGameplay.ShowCurrentBuildingDialog(TargetBuildingEntity);
```

**Comment by moepi2k** [comment]
line 236 is -> #region Sync data changes callback. and is not in a function

**Comment by insthync** [comment]
Your source codes may different from mine, Can you find ShowCurrentBuildingDialog function?

**Comment by moepi2k** [comment]
yes the ShowCurrentBuildingDialog function is on line 423

**Comment by insthync** [comment]
This was fixed in 1.54
